packages/blocks/code/package.json

```diff
@@ -19,7 +19,7 @@
     "serve": "block-scripts serve"
   },
   "dependencies": {
-    "blockprotocol": "0.0.7",
+    "blockprotocol": "0.0.8",
```

Comment:
Let's revert changes in these files because this diff is unrelated to this PR. We'll need to use an older version of BP and fixed React versions in the new package.

Reply:
Hmmm, same problem with some other block PRs, e.g. https://github.com/hashintel/hash/pull/515/files#diff-d335511879c31834c6c2251b8b14c7aacc0af94d41c993a2f31432910379e55b Maybe we should bump these deps everywhere to reduce the diffs. I'll do it.

Reply:
https://github.com/hashintel/hash/pull/537

Reply:
Thanks, I think `yarn fix` made those changes automatically.